modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/meetings/portlet/MeetingsPortlet.java

```diff
@@ -151,6 +152,55 @@ public void deletePowwowMeeting(
 		writeJSON(actionRequest, actionResponse, jsonObject);
 	}
 
+	public void deleteOccurrence(
+			ActionRequest actionRequest, ActionResponse actionResponse)
+		throws Exception {
+
+		ServiceContext serviceContext = ServiceContextFactory.getInstance(
+			PowwowMeeting.class.getName(), actionRequest);
+		ThemeDisplay themeDisplay = (ThemeDisplay)actionRequest.getAttribute(
+			WebKeys.THEME_DISPLAY);
+
+		JSONObject jsonObject = JSONFactoryUtil.createJSONObject();
+		long occurrenceId = ParamUtil.getLong(actionRequest, "occurrenceId", 0);
+
+		try {
+			if (occurrenceId > 0) {
+
+				PowwowMeetingOccurrence occurrence =
+					PowwowMeetingOccurrenceLocalServiceUtil
+						.fetchPowwowMeetingOccurrence(occurrenceId);
+
+				PowwowMeetingOccurrenceServiceUtil
+					.updateOccurrenceStatus(
+						occurrence.getPowwowMeetingId(),
+						occurrenceId, OccurrenceStatus.DELETE);
+
+				if (occurrence.getCalendarBookingId() > 0) {
+					CalendarBooking calendarBooking =
+						CalendarBookingLocalServiceUtil
+							.fetchCalendarBooking(occurrence.getCalendarBookingId());
+
+					calendarBooking.setStatus(CalendarBookingWorkflowConstants.STATUS_IN_TRASH);
```

Comment:
consider `STATUS_IN_TRASH` or `STATUS_INACTIVE`? just wondering, as long as you feel it's correct.